TodoList-Capstone/ViewModels/ListViewModel.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  listViewModel.swift
+//  TodoList-Capstone
+//
+//  Created by Om Prakash Singh on 09/11/24.
+//
+
+import Foundation
+
+class ListViewModel: ObservableObject {
+    @Published var items: [ItemModel] = []
+
+    func getItems() {
+        let newItems = [ItemModel(title: "First title", isCompleted: true),
+                        ItemModel(title: "Second title", isCompleted: false)]
+        items.append(contentsOf: newItems)
+    }
+
+    init() {
```

Comment:
I like to put the init() right after the properties are declared. This helps keep the object's construction separate from its functionality.

Reply:
got it , will do.